src/platform/ESP32/ConnectivityManagerImpl_Ethernet.cpp

```diff
@@ -51,6 +51,8 @@ namespace DeviceLayer {
 
 CHIP_ERROR ConnectivityManagerImpl::InitEthernet()
 {
+    // Initialize TCP/IP network interface (should be called only once in application)
+    ESP_ERROR_CHECK(esp_netif_init());
```

Comment:
I didn't understand the comment. If it should be called only once in the application, then why the call is here? If not, why the comment is here :-)

Reply:
I had added the call here assuming that the InitEthernet method will be called only once as currently there is no separate initialization which can be implemented, should I remove the comment?

Reply:
I think you should update this comment saying that: this is required for all Ethernet boards

Reply:
Added this change